leenlab2/reminiscence
# Save screenshot of tv to printout

# Description

Screenshot of what is currently playing on TV is printed to a printout that is placed in the player's hand.

### Type of change






New Feature
## How to Integrate

Add Print Screenshot Script to Video Player game object. The rest are just code changes.

# How Has This Been Tested?

- Open SaveScreenShotFunctionality scene
- Put the tape into the TV
- Open the TV and play to some random frame in the video
- Press the print button
- A new printout of what is currently on the TV should be placed in the player's hand. You can keep printing new ones if you would like. Walk around with the printout, drop and pick it up a few times to ensure it works as a typical intractable object.

# Checklists
## Integration Checklist

## Review Checklist
